assets/queries/cloudFormation/iam_access_keys_active_for_root/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "IAM_Access_Keys_Associated_To_Root_Account",
+  "queryName": "IAM Access Keys Associated To Root Account",
+  "severity": "HIGH",
+  "category": "Identity and Access Management",
+  "descriptionText": "Check if the root user has any access keys associated to it.",
+  "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-iam-accesskey.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline